libs/shared/ui/src/lib/components/icon/icons-status/build-error.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { type IconSVGProps } from '../icon'
+
+export const BuildErrorIcon = forwardRef<SVGSVGElement, IconSVGProps>(function BuildErrorIcon(
```

Comment:
```suggestion
export const BuildErrorIcon = forwardRef<SVGElement, IconSVGProps>(function BuildErrorIcon(
```
https://developer.mozilla.org/en-US/docs/Web/API/SVGSVGElement
https://developer.mozilla.org/en-US/docs/Web/API/SVGElement

Reply:
Already try but I can't do that 

<img width="815" alt="Capture d’écran 2023-09-19 à 14 26 00" src="https://github.com/Qovery/console/assets/533928/bb9b0ffa-1639-4a86-aee9-2a5b32d88b66">